yelp_package/Makefile

```diff
@@ -26,7 +26,9 @@ build_lucid_docker:
 	[ -d ../dist ] || mkdir ../dist
 	cd dockerfiles/lucid/ && docker build -t "paastatools_lucid_container" .
 package_lucid_yelp: build_lucid_docker
-	cp setup.py ../; cp requirements.txt ../
+	cp ../requirements.txt requirements.txt.bak
+	cat requirements-extras.txt >> ../requirements.txt
+	mv requirements.txt.bak ../requirements.txt
```

Comment:
I'm pretty sure the result of this is to clobber the changes you just made to ../requirements.txt. I'm pretty lost about what is happening. Also we don't need setup.py in .. anymore?